spark/src/test/scala/org/apache/comet/CometFuzzTestSuite.scala

```diff
@@ -206,10 +206,15 @@ class CometFuzzTestSuite extends CometTestBase with AdaptiveSparkPlanHelper {
   }
 
   private def testParquetTemporalTypes(
-      outputTimestampType: ParquetOutputTimestampType.Value): Unit = {
+      outputTimestampType: ParquetOutputTimestampType.Value,
+      generateArray: Boolean = true,
```

Comment:
should we also generate maps?

Reply:
It would be good to start testing maps as well in the fuzz suite. Some of the existing tests cannot work with map types though (because maps don't implementing ordering). We may need to add new map-specific tests.

Reply:
For this PR I'd just like to match existing functionality. We should open an issue to add map coverage to the CometFuzzTestSuite.

Reply:
I filed an issue: https://github.com/apache/datafusion-comet/issues/1665

Reply:
Filed https://github.com/apache/datafusion-comet/issues/1666